data/gegno/gegno intro missions.txt

```diff
@@ -432,6 +433,312 @@ mission "Giaru Gegno: Revisit"
 				decline
 
 
+mission "Suspicious Scin"
```

Comment:
Should there be an option to decline this mission? It feels to me that the `dialog`s when entering systems could get annoying for uninterested players. Have you thought about whether you want to rely on just the abort button and not offer a `decline` here?